app/index.js

```diff
@@ -16,10 +16,10 @@ function camelcase(name) {
     return words.join("");
 }
 
-module.exports = yeoman.generators.Base.extend({
-    prompting: function () {
-        var done = this.async();
```

Comment:
Don't use this anymore - it crashes out at the end of the first method if you do - use the new `await` keyword